tests/utils/test_databricks_utils.py

```diff
@@ -127,11 +127,27 @@ def test_get_model_dependency_oauth_token_model_serving_throws():
         databricks_utils.get_model_dependency_oauth_token()
 
 
-def test_databricks_params_model_serving_oauth_cache(monkeypatch, oauth_file):
+def test_databricks_params_model_serving_oauth_cache_databricks_env_var(monkeypatch, oauth_file):
```

Comment:
```suggestion
@pytest.mark.parametrize("env_var", [...])
def test_databricks_params_model_serving_oauth_cache_databricks_env_var(monkeypatch, oauth_file, env_var):
```

can we `parametrize` the env var name?